include/lsst/meas/base/NaiveCentroid.h

```diff
@@ -41,7 +41,9 @@ namespace base {
 /**
  *  @brief A C++ control class to handle NaiveCentroidAlgorithm's configuration
  */
-class NaiveCentroidControl {
+class [[deprecated(
+        "This algorithm and its control class are deprecated and will be removed after "
+        "v27.")]] NaiveCentroidControl {
```

Comment:
It'd be best to suppress these warnings in the pybind11 module so we don't see them when compiling that also-deprecated code.  The preprocessor directives to do that can be found in the [dev guide just above your new paragraph](https://developer.lsst.io/stack/deprecating-interfaces.html#c-deprecation).

Reply:
Thanks. I had to put the pragma around the entire pybind11 block in the namespace, but that did take care of the compiler warnings.